#!/usr/bin/env python
PACKAGE = "object_tracking_2d_ros"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()
#       Name		Type      	Level	Description 	Default Min   Max
gen.add("tf_x",		double_t,	0, 	"position.x", 	-5,  0, 5)
gen.add("tf_y",		double_t,	0, 	"position.y", 	-5,  0, 5)
gen.add("tf_z",		double_t,	0, 	"position.x",  	0,  1, 5)

gen.add("rot_x",	double_t,	0, 	"orientation.x", 0,  0, 2*math.pi)
gen.add("rot_y",	double_t,	0, 	"orientation.x", 0,  0, 2*math.pi)
gen.add("rot_z",	double_t,	0, 	"orientation.x", 0,  0, 2*math.pi)
gen.add("rot_w",	double_t,	0, 	"orientation.x", 0,  1, 2*math.pi)

exit(gen.generate(PACKAGE, "object_tracking_2d_ros", "object_tracking_2d_ros_init"))
